### PR TITLE
e2e node: make possible to add additional kernel arguments

### PR DESCRIPTION
**Pull request by cynepco3hahue**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Add an option to configure additional kernel arguments during
the setup of the e2e node environment.

The example:
```
cos-stable1:
image_family: cos-89-lts # docker v19.03.6, deprecated after 2021-06-24
project: cos-cloud
metadata: "user-data<test/e2e_node/jenkins/cos-init-live-restore.yaml,gci-update-strategy=update_disabled"
kernel_arguments:
- "numa=fake=2"
machine: n1-standard-4
```

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: test/e2e_node/runner/remote/run_remote.go [hunk 1]
```diff
@@ -702,9 +706,84 @@ func createInstance(imageConfig *internalGCEImage) (string, error) {
...
+	if err := rebootInstance(instance); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func rebootInstance(instance *compute.Instance) error {
+	// wait until the instance will not response to SSH
+	klog.Info("Reboot the node and wait for instance not to be available via SSH")
```
Comment (matthyx) [inline test/e2e_node/runner/remote/run_remote.go:762]: nit: ... for instance to stop responding via SSH

File: test/e2e_node/runner/remote/run_remote.go [hunk 2]
```diff
@@ -702,9 +706,84 @@ func createInstance(imageConfig *internalGCEImage) (string, error) {
...
+	}
+
+	if err := rebootInstance(instance); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func rebootInstance(instance *compute.Instance) error {
```
Comment (matthyx) [inline test/e2e_node/runner/remote/run_remote.go:760]: I hope this is not too flacky...

**Review by matthyx — APPROVED** [review]

**Review by mrunalp — APPROVED** [review]

**Pull request merged**